pkg/cmd/policy_rule_matcher_test.go

```diff
@@ -15,15 +16,20 @@ func TestMatcher_MatchesRole(t *testing.T) {
 		Rules: []rbac.PolicyRule{
 			{
 				Verbs:     []string{"get", "list"},
+				APIGroups: []string{""},
```

Comment:
Given you've gone to the trouble of adding (empty) APIGroups into these tests I wonder if it's worth have a case that is non-empty, more to catch regressions / document what is supposed to happen than for code coverage. Maybe not really necessary because you have tests for matching against individual rules in `TestMatcher_matches`. Wdyt?

Reply:
Definitely worth it. Thanks! I've added the case with `deployments.extensions`.